docs/.vitepress/data/guides.ts

```diff
@@ -212,6 +212,19 @@ export const sections = {
 					},
 				],
 			},
+			{
+				title: 'Modules',
+				items: [
+					{
+						display: 'Create a Custom Portal Module',
+						path: '/guides/extensions/modules-build-portal',
```

Comment:
Of course this requires the file to be renamed too
```suggestion
						display: 'Create a Custom Landing Page Module',
						path: '/guides/extensions/modules-build-landing-page',
```